neural_compressor/torch/algorithms/weight_only/rtn.py

```diff
@@ -21,34 +21,33 @@
 
 import torch
 
-from neural_compressor.torch.utils import logger
-from neural_compressor.torch.utils.utility import set_module
+from neural_compressor.torch.utils import logger, set_module
```

Comment:
Kindly reminder, we have already enabled some examples using 3.x APIs on https://github.com/intel/neural-compressor/pull/1498. Please do some extension tests if possible.

Reply:
OK